package.json

```diff
@@ -48,7 +48,8 @@
 		"tailwindcss": "^4.0.0",
 		"typescript": "^5.0.0",
 		"typescript-eslint": "^8.20.0",
-		"vite": "^6.2.6"
+		"vite": "^6.2.6",
+		"zod": "^4.0.17"
```

Comment:
Required to avoid `Error: Missing ./v4/core specifier in zod package` caused by import of `arktypeClient` in `src/routes/(main)/auth/register/+page.svelte`. Even though zod isn't used, it's a peer dependency in `sveltekit-superforms`.

Related: https://github.com/ciscoheat/sveltekit-superforms/issues/351